docs/specifications/tests/Zone-TP/zone10.md

```diff
@@ -0,0 +1,94 @@
+# ZONE10: No multipe SOA records
+
+
+## Test case identifier
+**ZONE10**
+
+
+## Objective
+
+The SOA record is crucial for the DNS zone and "exactly one SOA RR should
+be present at the top of the zone" ([RFC 1035][RFC 1035#5.2], section 5.2).
+This test case will verify that the zone of the domain to be tested return
+exactly one SOA record.
+
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+
+## Ordered description of steps to be taken to execute the test case
+
+1. Obtain the set of name server IP addresses using [Method4] and [Method5]
+   ("Name Server IP").
+
+2. Create a SOA query for the apex of the *Child Zone* with RD flag unset.
+
+3. For each name server in *Name Server IP* do:
+   1. Send the SOA query over UDP to the name server.
+   2. If the name server does not respond with a DNS response, then
+      output *[NO_RESPONSE]*.
+   3. Else, if the DNS response does not include a SOA record in the
+      answer section, then output *[NO_RESPONSE_SOA_QUERY]*.
+   4. Else, if the SOA record or records in the answer section has
+      an owner name same as *Child Zone*, then output
+      *[WRONG_SOA]*.
+   4. Else, if the DNS response includes multipel SOA records in the
+      answer section, then output *[MULTIPLE_SOA]*.
+
+4. If no message is outputted for any server, then output *[ONE_SOA]*.
```

Comment:
Not clear what does ONE_SOA is

Reply:
The message means that all servers have exactly one SOA record for the zone.